.eslintrc.json

```diff
@@ -1,27 +1,12 @@
 {
```

Comment:
I don't think this should be in the commit, everyone's lint is different, not relevant to your feature. Maybe add to gitignore then a new commit?